src/headers/SiteHeader.stories.tsx

```diff
@@ -9,14 +9,22 @@ export default {
 export const basic = () => (
   <SiteHeader
     homeURL={"/"}
-    // languages={[{ label: "English", onClick: () => console.log("Clicked English"), active: true }]}
```

Comment:
Are these intentionally commented out?

Reply:
Was just removing the commented lines to clean up, afaik I don't think they were left commented on purpose